New: email configuration: configurable recipient address (#4403)
owncloud/docs: modules/admin_manual/pages/configuration/server/email_configuration.adoc

image:configuration/server/email-configuration/smtp-config-smtp.png[image]

Your changes are saved immediately.

> You must enter a valid email address where a test email will be sent to. This field is mandatory and will be prefilled with the logged-in admin's email if set, but can be changed to any other valid email address. Note that this test recipient address can't be set via a config value in `config.php`. Click the btn:[Send Email] button to test your configuration. This sends a test message to the email address you configured. The test message says:

Reviewer: ```suggestion
You must enter a valid email address to which a test email will be sent. This field is mandatory and will be prefilled with the logged-in admin's email if set, but can be changed to any other valid email address. Note that this test recipient address can't be set via a config value in `config.php`. Click the btn:[Send Email] button to test your configuration. This sends a test message to the email address you configured. The test message says:
```

image:configuration/server/email-configuration/smtp-config-php.png[image]

> Same as with the SMPT configuration above, you must enter an email address where a test email will be sent to before testing the configuration. For details see the section above.

Reviewer: ```suggestion
Same as with the SMPT configuration above, you must enter an email address to which a test email will be sent before testing the configuration. For details see the section above.
```

> NOTE: PHP mode uses your local `sendmail` binary. and any drop-in Sendmail replacement such as Postfix, Exim, or Courier. All of these include a `sendmail` binary, and are freely-interchangeable. Use this if you want to use `php.ini` to control some of your mail server functions, such as setting _paths_, _headers_, or passing extra command options to the `sendmail` binary. These vary according to which server you are using, so consult your server’s documentation to see what your options are.

Reviewer: ```suggestion
NOTE: PHP mode uses your local `sendmail` binary and any drop-in Sendmail replacement such as Postfix, Exim, or Courier. All of these include a `sendmail` binary, and are freely interchangeable. Use this if you want to use `php.ini` to control some of your mail server functions, such as setting _paths_, _headers_, or passing extra command options to the `sendmail` binary. These vary depending on which server you are using, so consult your server’s documentation to see what your options are.
```
